lib/datadog/tracing/contrib/action_pack/action_dispatch/patcher.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+require_relative '../../patcher'
+require_relative 'instrumentation'
+
+module Datadog
+  module Tracing
+    module Contrib
+      module ActionPack
+        module ActionDispatch
+          # Patcher for ActionController components
+          module Patcher
+            include Contrib::Patcher
+
+            module_function
+
+            def target_version
+              Integration.version
+            end
+
+            def patch
+              if ::ActionPack.gem_version >= Gem::Version.new(7.1)
```

Comment:
Would `'7.1'` not be more correct usage?

Reply:
you are right, it is supposed to work with a string

Reply:
fixed